mypyc/test-data/run-lists.test

```diff
@@ -537,3 +537,33 @@ def test_sorted() -> None:
     assert sorted((2, 1, 3)) == res
     assert sorted({2, 1, 3}) == res
     assert sorted({2: "", 1: "", 3: ""}) == res
+
+[case testIsInstance]
+from copysubclass import subc
+def test() -> None:
+    assert isinstance([], list)
+    assert isinstance([1,2,3], list)
+    assert isinstance(['a','b'], list)
+    assert isinstance(subc(), list)
+    assert isinstance(subc([1,2,3]), list)
+    assert isinstance(subc(['a','b']), list)
+
+    assert not isinstance({}, list)
+    assert not isinstance((), list)
+    assert not isinstance((1,2,3), list)
+    assert not isinstance(('a','b'), list)
+    assert not isinstance(1, list)
+    assert not isinstance('a', list)
+
+[file copysubclass.py]
+from typing import Any
+class subc(list[Any]):
+    pass
+
+[case testUserDefinedList]
+class list:
+    pass
+
+def test() -> None:
```

Comment:
I don't think this gets executed, since the test runner looks for a `test_` prefix. Also, can you merge the new test cases under one `[case ...]` and add multiple `def test_...` methods for sub test cases? Each top-level test case has significant constant overhead, so having a few, bigger test cases makes the test suite run faster.